blocks/large-promo-block/features/large-promo/default.jsx

```diff
@@ -99,12 +139,148 @@ const LargePromoItem = ({ customFields, arcSite }) => {
     }`,
 		}) || null;
 
-	return <LargePromoPresentation content={content} {...customFields} />;
+	// const { id } = useFusionContext();
+	const { editableContent, searchableField } = useEditableContent();
+	const { registerSuccessEvent } = useComponentContext();
+	const {
+		dateLocalization: { language, timeZone, dateTimeFormat } = {
+			language: "en",
+			timeZone: "GMT",
+			dateFormat: "LLLL d, yyyy 'at' K:m bbbb z",
+		},
+	} = getProperties(arcSite);
+	const phrases = getTranslatedPhrases(getProperties(arcSite).locale || "en");
+	const bylineNodes = formatAuthors(
+		content?.credits?.by,
+		phrases.t("global.byline-block-and-text")
```

Comment:
let's make sure that the phrases and bylines match the intl used 

byline-block.and-text